soroban-sdk/src/storage.rs

```diff
@@ -291,6 +294,42 @@ impl Persistent {
     }
 }
 
+#[cfg(any(test, feature = "testutils"))]
+#[cfg_attr(feature = "docs", doc(cfg(feature = "testutils")))]
+impl testutils::storage::Persistent for Persistent {
```

Comment:
Can we create a helper to de-duplicate persistent and temp code? IIUC the only difference is in `durability` filter.

Reply:
Yup, although value seems mediocre and may result in harder to maintain code down the line.

Reply:
I don't expect persistent and temp storage interfaces to ever diverge, but up to you really.

Reply:
380eae4

Reply:
That worked well, thanks 👏🏻.